### PR TITLE
Alerts pt2

### PR DESCRIPTION
**Pull request by dgrammatiko**
Added 
- Animations for entry/exit
- Dropping SCSS for plain CSS
- Using CSS Vars to control the layout/style of alerts (dev friendlier than scss)
- update docs
- update changelog

### CODE REVIEW

File: src/scss/alert/alert.scss [hunk 1]
```diff
@@ -1,121 +1,118 @@
...
-  min-width: 250px;
-  padding: $alert-padding-y $alert-padding-x;
-  margin-bottom: $alert-margin-bottom;
-  border: $alert-border-width solid transparent;
-  border-radius: $alert-border-radius;
-  opacity: 0;
-  animation: jui-alert-fade-in linear .5s;
-  // animation: jui-alert-fade-out linear .5s;
-
-  @media screen and (prefers-reduced-motion: reduce) {
```
Comment (wilsonge) [inline src/scss/alert/alert.scss:62]: With the keyframes animations do we need to still have this `prefers-reduced-motion: reduce` flag code? I'm not sure on how this flag treats animations vs transitions.

Comment (dgrammatiko) [reply src/scss/alert/alert.scss:62]: Usually this should be on the template, global css (eg https://github.com/joomla/joomla-cms/blob/a5183489f0b2e2fc77741c217e5933b24de2a7c1/administrator/templates/atum/scss/template.scss#L70-L82)
but I can add it here scoped: 
```css
@media (prefers-reduced-motion: reduce) {
  joomla-alert {
    animation-duration: 1ms !important;
  }
}
```

Comment (wilsonge) [reply src/scss/alert/alert.scss:62]: I think that would make sense. In my opinion it should be self-contained. It's not like it's a media query specific to the CMS.

```
@media (prefers-reduced-motion: reduce) {
  joomla-alert {
    animation: none;
    opacity: 1;
  }
}
```

would be even better wouldn't it?

Comment (dgrammatiko) [reply src/scss/alert/alert.scss:62]: ` animation: none;` That won't work, both the events `shown` and `closed` are fired at the end of the animation so disabling the animation with a none value means that we have to manually fire those events (basically what the old code was doing). Anyways I already added the single line needed here with https://github.com/joomla-projects/custom-elements/pull/179/commits/19ded6e4e9482050834f23f525c86aa6d24eb5f5

**Comment by dgrammatiko** [comment]
Thanks, could you release a 0.1.0 version so I can update the PR in the main repo?

**Comment by wilsonge** [comment]
Done!

**Pull request merged**